Libplanet/Net/SwarmOptions.cs

```diff
@@ -128,7 +128,7 @@ public enum TransportType : byte
         /// for the configured lifespan, <see cref="Swarm{T}"/> pulls new blocks from neighbor
         /// peers.
         /// </summary>
-        public TimeSpan TipLifespan { get; set; } = TimeSpan.FromSeconds(30);
+        public TimeSpan TipLifespan { get; set; } = TimeSpan.FromSeconds(60);
```

Comment:
This change of the default value should be recorded in the changelog.

Reply:
Done